scripts/docker/build.cmd

```diff
@@ -33,10 +33,12 @@ cd %APP_HOME%
     xcopy /s src\*       out\docker\src\
     xcopy /s public\*    out\docker\public\
 
+    copy public\webui-config.js                     out\docker\
```

Comment:
Line 34 should be already copying webui-config.js, why do you need another copy?

Reply:
yes not required!